src/Sulu/Bundle/MarkupBundle/Markup/HtmlTagExtractor.php

```diff
@@ -63,7 +63,7 @@ public function extract($html)
             $sortedTags[$name][$tag] = array_filter(array_merge($attributes, ['content' => $content]));
         }
 
-        return $sortedTags;
+        return [$this->namespace => $sortedTags];
```

Comment:
but then we have to iterate over this array to find all tags with the namespace. this would make the rest of the process "slower"